docker/README.md

```diff
@@ -118,12 +118,22 @@ It becomes fairly easy to start Kill Bill locally on your laptop. For example le
   ```
   echo "set global binlog_format = 'ROW'" | mysql -h $(docker-machine ip default) -uroot -p
   ```
-  And then create the database `killbill_0_16_0` and add the DDLs:
+  And then add the DDLs:
 
   * Kill Bill [DDL](http://docs.killbill.io/0.16/ddl.sql)
+
+    ```curl http://docs.killbill.io/0.16/ddl.sql | mysql -h $(docker-machine ip default) -Dkillbill -uroot -p```
```

Comment:
Good call. And I changed `-p` to `-proot` so it won't prompt for a password at all.